libnavigation-core/src/main/java/com/mapbox/navigation/core/trip/session/MapboxTripSession.kt

```diff
@@ -161,6 +161,7 @@ internal class MapboxTripSession(
             if (fallbackVersionsObservers.isNotEmpty()) {
                 navigator.setFallbackVersionsObserver(nativeFallbackVersionsObserver)
             }
+            navigator.addNavigatorObserver(navigatorObserver)
```

Comment:
scenario: 
- `MapboxNavigation` is created
- navigator switched to offline/online more
- start TripSession

in this case, `navigatorObserver` is set to navigator twice. Is it expected? should we check if a session is started?